config/initializers/assets.rb

```diff
@@ -12,3 +12,5 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
+Rails.application.config.assets.precompile += %w( cocoon.js )
+Rails.application.config.assets.precompile += %w( add_question.js )
```

Comment:
- duplicate, use: `Rails.application.config.assets.precompile += %w( add_question.js  cocoon.js )` instead.
- why do we need precompile `add_question.js` while we already included it in view file?